### PR TITLE
fixes improper string interpolation in relative dates causing 'false' to be present when withoutSuffix is disabled

### LINKED ISSUE #7188: Patient details page, relative date time having "false" in text
**Expected behavior**
"false" should not be present in relative date time

**Screenshots**
image

cc: @nihal467

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Fixes #7188

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
image

add a space between ago and at @rithviknishad

File: src/Utils/utils.ts [hunk 1]
```diff
@@ -101,9 +101,9 @@ export const formatTime = (date: DateLike, format = TIME_FORMAT) =>
 
 export const relativeDate = (date: DateLike, withoutSuffix = false) => {
   const obj = dayjs(date);
-  return `${obj.fromNow(withoutSuffix)} ${
-    withoutSuffix && "ago"
-  } at ${obj.format(TIME_FORMAT)}`;
+  return `${obj.fromNow(withoutSuffix)}${
+    withoutSuffix ? " ago " : ""
+  }at ${obj.format(TIME_FORMAT)}`;
```
Comment (Ashesh3) [inline src/Utils/utils.ts]:
```suggestion
  } at ${obj.format(TIME_FORMAT)}`;
```

**Comment by nihal467** [comment]
LGTM

File: src/Utils/utils.ts [hunk 2]
```diff
@@ -101,8 +101,8 @@ export const formatTime = (date: DateLike, format = TIME_FORMAT) =>
 
 export const relativeDate = (date: DateLike, withoutSuffix = false) => {
   const obj = dayjs(date);
-  return `${obj.fromNow(withoutSuffix)} ${
-    withoutSuffix && "ago"
+  return `${obj.fromNow(withoutSuffix)}${
+    withoutSuffix ? " ago " : ""
```
Comment (khavinshankar) [inline src/Utils/utils.ts:105]:
```suggestion
    withoutSuffix ? " ago" : ""
```

**Review by khavinshankar — COMMENTED** [review]
just a very minor change

File: src/Utils/utils.ts (hunk 2 above)
Comment (Ashesh3) [reply src/Utils/utils.ts:105]: Change LGTM

**Review by khavinshankar — APPROVED** [review]

File: src/Utils/utils.ts (hunk 2 above)
Comment (Ashesh3) [reply src/Utils/utils.ts:105]: @khavinshankar we did not commit this change before merging 😅

**Pull request merged**